### ISSUE TITLE
[Python - sphinx] Travis (Mac OS) ImportError: cannot import name 'main' from 'sphinx' 

### ISSUE THREAD

**Issue by VincentRouvreau** [GUDHI/gudhi-devel#77]
```
$ ctest -R sphinx_py_test -V
UpdateCTestConfiguration  from :/Users/travis/build/GUDHI/gudhi-devel/build/DartConfiguration.tcl
UpdateCTestConfiguration  from :/Users/travis/build/GUDHI/gudhi-devel/build/DartConfiguration.tcl
Test project /Users/travis/build/GUDHI/gudhi-devel/build
Constructing a list of tests
Done constructing a list of tests
Updating test list for fixtures
Added 0 tests to meet fixture requirements
Checking test dependency graph...
Checking test dependency graph end
test 30
    Start 30: sphinx_py_test
30: Test command: /usr/local/Cellar/cmake/3.14.2/bin/cmake "-E" "env" "PYTHONPATH=/Users/travis/build/GUDHI/gudhi-devel/build/src/cython" "/usr/local/bin/python3" "/Users/travis/build/GUDHI/gudhi-devel/src/cython/doc/python3-sphinx-build.py" "-b" "doctest" "/Users/travis/build/GUDHI/gudhi-devel/src/cython/doc" "/Users/travis/build/GUDHI/gudhi-devel/build/src/cython/doctest"
30: Test timeout computed to be: 10000000
30: Traceback (most recent call last):
30:   File "/Users/travis/build/GUDHI/gudhi-devel/src/cython/doc/python3-sphinx-build.py", line 8, in <module>
30:     from sphinx import main
30: ImportError: cannot import name 'main' from 'sphinx' (/Users/travis/Library/Python/3.7/lib/python/site-packages/sphinx/__init__.py)
1/1 Test #30: sphinx_py_test ...................***Failed    0.20 sec
0% tests passed, 1 tests failed out of 1
```

**Comment by mglisse** [comment]
I think you've fixed that one?

**Comment by VincentRouvreau** [comment]
Oh yes ! Fixed by #329
